timesync/peersync/round.go

```diff
@@ -0,0 +1,49 @@
+package peersync
+
+import (
+	"sort"
+	"time"
+)
+
+type timedResponse struct {
+	Response
+	receiveTimestamp int64
+}
+
+type round struct {
+	ID                uint64
+	Timestamp         int64
+	RequiredResponses int
+	responses         []timedResponse
+}
+
+func (r *round) AddResponse(resp Response, timestamp int64) {
```

Comment:
can `AddResponse` and `Ready` br called at the same time? should this struct be thread safe?

Reply:
it doesn't need to be thread safe